web/modules/custom/portland_migrations/migrations/auditor_blog.yml

```diff
@@ -0,0 +1,38 @@
+id: auditor_blog
+label: 'Auditor Blog Import'
+migration_group: auditor
```

Comment:
The migration_group is used to link together all migrations that are related to each other and should be run together. So for this migration, it should probably be set to 'auditor_blog'.